relnotes/verify.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to Sociomantic assert/enforce policies.
```

Comment:
Yes, neptune is 100% about out usage of SemVer and related tools. General coding guidelines don't fit there.